### DISCUSSION TITLE
Importing markdown files

### DISCUSSION THREAD

**Discussion by oriongonza** [nvim-neorg/neorg#1117] [General]
Is there a way to transform .md files into .norg files?

**Comment by jonboh** [comment]
I've made a basic application to transform obsidian markdown files into Neorg. I doesn't implement all possible transformations for the markdown syntax but it should be enough to transfer a large set of notes so that links and basic style is transformed.
https://github.com/jonboh/obsidian2neorg

I've used it to port my Obsidian vault to Neorg. It uses regex to make the transformations, if it isn't enough for your use case you might be able to extend it from there.

**Comment by pysan3** [comment]
I've implemented a pandoc to norg converter: https://github.com/pysan3/nim_norg

Use pandoc to parse your markdown and output to json format. Pass that json via file name (`--input`) or stdin and you'll get a norg output. 

Feel free to send issues if you got some bugs as it is still in very alpha.

**Reply by pysan3** [reply]
I've updated my converter since then and got some level of testing done as well. 

Could you test it on your pile of notes? @dev-ardi 

If you are converting from obsidian, don't forget to read the obsidian section.

**Reply by oriongonza** [reply]
It's working for me :)
